src/main/java/org/team1540/robot2024/util/PhoenixTimeSyncSignalRefresher.java

```diff
@@ -0,0 +1,38 @@
+package org.team1540.robot2024.util;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.CANBus;
+import edu.wpi.first.wpilibj.Timer;
+import org.team1540.robot2024.Constants;
+
+import java.util.Arrays;
+
+import static org.team1540.robot2024.Constants.SwerveConfig.*;
+
+/**
+ * Helper class for refreshing CANivore TimeSynced status signals.
+ */
+public class PhoenixTimeSyncSignalRefresher {
+    private static BaseStatusSignal[] signals = new BaseStatusSignal[0];
+    private static final boolean isCANFD = CANBus.isNetworkFD(CAN_BUS);
+
+    private static double lastRefreshTimestamp = 0;
+
+    /**
+     * Registers status signals to be refreshed when {@link PhoenixTimeSyncSignalRefresher#refreshSignals()}is called
+     */
+    public static void registerSignals(BaseStatusSignal... newSignals) {
```

Comment:
wait wait wait wait Alvin no why is this static? 


...We have two CAN busses.

Reply:
The signal refresher is only meant to be used with timesynced signals on a CANivore bus, since those are the only ones that care about being refreshed all at the same time. I could potentially change it so that it can refresh all signals, if that's cleaner to you.

Reply:
Alvin using a static setup for this is incredibly unclear, regardless of what you intended it to be used by. If this is only for odometry, make an instance and pass it to all of the modules.